net/proxy_resolution/proxy_config_service_tor.cc

```diff
@@ -140,10 +141,12 @@ std::string ProxyConfigServiceTor::CircuitIsolationKey(const GURL& url) {
   url::Origin url_origin = url::Origin::Create(url);
   net::NetworkIsolationKey network_isolation_key(url_origin, url_origin);
```

Comment:
We should also migrate the old constructor use which will be removed soon
https://chromium.googlesource.com/chromium/src.git/+/4807a9ac25ec2b53f5e24750b67010817748a8f7%5E%21/

Reply:
Updated in 60feea146200d54f57ae75e34102f2d86f708a6d